compiler/tests-compiler/lambda_lifting.ml

```diff
@@ -22,17 +22,23 @@ Printf.printf "%d\n" (f 3)
     {|
     (function(globalThis){
        "use strict";
+       var runtime = globalThis.jsoo_runtime;
+       function caml_call2(f, a0, a1){
+        return (f.l >= 0 ? f.l : f.l = f.length) == 2
+                ? f(a0, a1)
+                : runtime.caml_call_gen(f, [a0, a1]);
+       }
+       runtime.caml_initialize_fiber_stack();
        var
-        runtime = globalThis.jsoo_runtime,
         global_data = runtime.caml_get_global_data(),
         Stdlib_Printf = global_data.Stdlib__Printf,
-        _b_ =
+        _e_ =
```

Comment:
I'm surprised by this diff, do you understand why it's here ?

Reply:
Isn’it because the generated program contains more variables, so all generated identifier names are “shifted”? And this effect is only visible on identifiers which do not correspond to an OCaml variable.

Reply:
This is only a hypothesis; to be sure I would need to fully understand how these identifiers are generated.